public/components/main/report_details/report_details.tsx

```diff
@@ -35,6 +35,7 @@ import { ShareModal } from './share_modal/share_modal';
 
 interface ReportDetailsRouteProps {
   reportId: string;
+  httpClient: any;
```

Comment:
Why is this `any` instead of the core start type?

Reply:
Good catch- will change/see if this props interface can be removed